README.md

```diff
@@ -32,38 +32,41 @@ cd lsk-faucet
 ```
 
 2. Bundle Frontend web with Vite
+
+**NOTE**: Please make sure to replace token icon at `web/public/token.png` with the specific ERC20 token icon.
+
 ```bash
-go generate
+make build-frontend
 ```
 
 3. Build Go project 
 ```bash
-go build -o lsk-faucet
+make build-backend
 ```
 
 ## Usage
 
 **Use private key to fund users**
 
 ```bash
-./lsk-faucet -httpport 8080 -wallet.provider http://localhost:8545 -wallet.privkey privkey
+make run FLAGS="-httpport 8080 -wallet.provider http://localhost:8545 -wallet.privkey privkey"
 ```
 
 **Use keystore to fund users**
 
 ```bash
-./lsk-faucet -httpport 8080 -wallet.provider http://localhost:8545 -wallet.keyjson keystore -wallet.keypass password.txt
+make run FLAGS="-httpport 8080 -wallet.provider http://localhost:8545 -wallet.keyjson keystore -wallet.keypass password.txt"
 ```
 
 ### Configuration
 Below is a list of environment variables that can be configured.
 
-- `WEB3_PROVIDER`: Endpoint for Lisk JSON-RPC connection.
+- `WEB3_PROVIDER`: RPC Endpoint to connect with the network.
 - `PRIVATE_KEY`: Private key hex to fund user requests with.
 - `KEYSTORE`: Keystore file to fund user requests with.
 - `HCAPTCHA_SITEKEY`: hCaptcha sitekey.
 - `HCAPTCHA_SECRET`: hCaptcha secret.
-- `LSK_TOKEN_ADDRESS`: Contract address of LSK token on the Lisk L2.
+- `ERC20_TOKEN_ADDRESS`: Contract address of ERC20 token on the above mentioned network, defaults to Lisk L2 token address.
```

Comment:
What do we mean by `above-mentioned network`? I do not see any specified networks. Can we please rephrase it to be more clear?
```suggestion
- `ERC20_TOKEN_ADDRESS`: Contract address of the ERC20 token on the above mentioned network, defaults to contract address for Lisk ERC20 tokens on Lisk Sepolia.
```